src/app/app-routing.module.ts

```diff
@@ -1,11 +1,13 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 import { PopularMoviesComponent } from './src/components/popular-movies/popular-movies.component';
+import { SearchPageComponent } from './src/components/search-page/search-page.component';
 import { TopRatedMoviesComponent } from './src/components/top-rated/top-rated-movies.component';
 
 const routes: Routes = [
   { path: 'top-rated', component: TopRatedMoviesComponent },
   { path: 'popular', component: PopularMoviesComponent },
+  { path: 'search-results', component: SearchPageComponent },
```

Comment:
you could put all path values in const and then reuse those consts in places like for example here
![image](https://user-images.githubusercontent.com/79691112/190324352-fd8895ae-cdc9-4304-b8f3-48b996a183d8.png)
could be `[routerLink]="SearchResults.path"`